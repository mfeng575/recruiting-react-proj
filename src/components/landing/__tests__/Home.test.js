import React from 'react';
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import toJson from 'enzyme-to-json';
import Home from '../Home';


describe('Home.test.js', () => {

    it('renders without crashing', () => {
        const wrapper = shallow(<Home/>);

        expect(wrapper).toHaveLength(1);

        expect(wrapper.find('Header')).toHaveLength(1);
        expect(wrapper.find('Footer')).toHaveLength(1);   

        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();    

    });
});
