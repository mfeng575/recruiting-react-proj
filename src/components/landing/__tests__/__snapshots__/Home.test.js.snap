// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.test.js renders without crashing 1`] = `
<div>
  <Header />
  <div
    className="container text-muted"
  >
    <Footer />
  </div>
</div>
`;
